### PR TITLE
Add docs for ansible network zuul

### PR DESCRIPTION
**Pull request by danielmellado**
This commit adds documentation regarding the system infrastructure in
place for ansible's zuul.

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/185
Signed-off-by: Daniel Mellado <dmellado@redhat.com>

### CODE REVIEW

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now 6bf3446c1f9de8be51067fe02f0b6dde9a4d7ea8

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now e8799a861697da86977ca931d234dad81e298e3c

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now d87ead111dc27115a321aaa6066bc40f8a538e09

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now e7479256cdc3eba99b17df628c84dec235873f6b

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now 05b555f8b10e2cf510d8db90bd33275dfe4c070c

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now 888eed1b6a93e0b39bc8ae3982ea8782ad1b2930

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now 2c98722587b21e7327ecd1af99308c39caa404eb

File: .zuul.d/nodesets.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (pabelanger) [inline .zuul.d/nodesets.yaml]: delete this, it is not needed

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now bbb5e41755ff66f8a32004afee17e406f23327a5

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now ec2a6fbe6b7fffbd308a200286c4527802b595d6

**Comment by danielmellado** [comment]
@pabelanger let's get this merged for now and iterate over it

**Comment by goneri** [comment]
@danielmellado could you please rebase the patch?

File: doc/source/ansible_zuul.rst [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+reference manual <https://docs.openstack.org/infra/zuul>`_.
+
+
+Ansible Zuul Hosts
+==================
+
+.. table:: **Ansible Zuul Hosts**
+
+  =========================================  ========  =================
+  Host                                       Provider  Role
```
Comment (goneri) [inline doc/source/ansible_zuul.rst:93]: It would be great to generate this list from the content of `ansible/hosts.yaml`.

**Comment by goneri** [comment]
I would like to land this patch. @pabelanger any objection?

File: .zuul.d/projects.yaml [hunk 3]
```diff
@@ -8,6 +8,7 @@
     gate:
       jobs:
         - windmill-config-tox-tools
+        - ansible-tox-docs
```
Comment (pabelanger) [inline .zuul.d/projects.yaml:11]: this also needs to be in check

**Comment by goneri** [comment]
Thank you @danielmellado.

**Pull request merged**